src/Entity/Admin.php

```diff
@@ -30,6 +30,8 @@ class Admin implements UserInterface
      */
     private ?string $password;
 
+    private ?string $plainPassword;
```

Comment:
I would simply put this field public, no need for a getter/setter

Reply:
Ok, good idea. I also changed the way the 'empty' admin is created for use by EasyAdmin, otherwise it complains about 'Type property must not be accessed before initialisation.